bin/automerge-prs

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+shopt -s nullglob
```

Comment:
If you're wondering how this fixes it...

This option controls what happens when `for x in {some glob}` finds no matches. By default, `x` will take on the value `{some glob}` (e.g. `./*.json`) and so `jq './*.json'` as a literal file path will fail. Enabling `nullglob` causes this to instead just not match anything (i.e. the `for` loop will never execute), kind of like any reasonable person would expect...

Reply:
What a horrible default behavior. :face_exhaling:

Reply:
Agreed. Though it might make a _little_ more sense for something like `sort *.txt > sorted.txt`. What should this do when there are not matches? Silently make an empty file? Erroring may seem a safer default in some cases.